### ISSUE TITLE
Generate RosSharp Messages

### ISSUE THREAD

**Issue by glennliu** [EricVoll/ros-sharp#2]
Hi @EricVoll 

I used your code of ROS# successful, which can be built on UWP platform. 

But the _RosSharp_ messages generation is quite different with the original code. Can you please explain a bit about message generation? How to generate a "sensor_msgs/PointCloud" message in your code?

Thanks

**Comment by EricVoll** [comment]
Hi @glennliu 
thanks for the question. I'll have a look at it in a few hours ok?
- Eric

**Comment by EricVoll** [comment]
Ok so on a first and quick look the case is the following:
I forked this repository from @dwhit who did the main bulk of the work. I made some final adjustments because on the newer Unity & HoloLens versions Ros# did not work anymore.
His fork is 101 commits behind the original master at (Siemens/RosSharp).
I'll may have time in the following weeks to synch this fork with the main repository, such that it provides all the functionality it does there.
